packages/@aws-cdk/aws-appmesh/test/test.mesh.ts

```diff
@@ -178,12 +172,9 @@ export = {
         });
 
         const testRouter = mesh.addVirtualRouter('test-router', {
-          listener: {
-            portMapping: {
-              port: 8080,
-              protocol: appmesh.Protocol.HTTP,
-            },
-          },
+          listeners: [
+            appmesh.VirtualRouterListener.httpVirtualRouterListener(),
+          ],
```

Comment:
Can we test the default at least once somewhere here?

Reply:
So it's still not tested 🙂. By "default", I mean `listeners` is not provided at all.

Reply:
I added a test for this in the `test.virtual-router.ts` file

Reply:
Went ahead and changed one of these here as well 👍

Reply:
Thanks!